manifests/00-crd-extension-console-link.yaml

```diff
@@ -53,4 +53,16 @@ spec:
             location:
               type: string
               description: Determines which dropdown menu the link is added to (HelpMenu, UserMenu)
-              pattern: ^(HelpMenu|UserMenu)$
+              pattern: ^(ApplicationMenu|HelpMenu|UserMenu)$
+            applicationMenu:
+              type: object
+              description: Object that holds application menu properties if the location is set to ApplicationMenu
+              required:
+                - section
+              properties:
+                section:
+                  type: string
+                  description: The section of the application menu in which the link should appear
+                imageURL:
```

Comment:
@benjaminapetersen @jhadvig do we need to add `+optional` to the property in openshift/api? I'm not aware of all the rules....

Reply:
Yes, we should.  I'll do a quick PR for that.